nxrm-ha/nx-values.yaml

```diff
@@ -0,0 +1,71 @@
+# Declare variables to be passed into your templates.
+namespaces:
```

Comment:
Was this file accidentally checked in?

Reply:
yes this file should be removed , was added for local testing.